lib/ui/painting.dart

```diff
@@ -5871,3 +5905,22 @@ Future<T> _futurize<T>(_Callbacker<T> callbacker) {
     throw Exception(error);
   return completer.future;
 }
+
+class PictureRasterizationException implements Exception {
```

Comment:
Does this need to be its own exception type, or can use use `Exception` for this?

Reply:
I want this to be something you can try to specifically catch, with the idea being that this API could potentially be grown to support more of an idea of what kind of exception was thrown (particularly if you tried to make a texture larger than the user's GPU supports)

Reply:
where would you put the catch though?

Reply:
I would give this a documentation comment at least describing the intent

Reply:
Done